src/components/Pressable/PressableWithFeedback.js

```diff
@@ -24,9 +25,34 @@ const PressableWithFeedbackDefaultProps = {
 
 const PressableWithFeedback = forwardRef((props, ref) => {
     const propsWithoutStyling = _.omit(props, omittedProps);
+    const [disabled, setDisabled] = useState(props.disabled);
+
+    useEffect(() => {
+        setDisabled(props.disabled);
+    }, [props.disabled]);
+
     return (
-        // eslint-disable-next-line react/jsx-props-no-spreading
-        <GenericPressable ref={ref} style={props.wrapperStyle} {...propsWithoutStyling}>
+        <GenericPressable
+            ref={ref}
+            style={props.wrapperStyle}
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...propsWithoutStyling}
+            disabled={disabled}
+            onPress={(e) => {
+                if (disabled) { return; }
```

Comment:
Is this required? If the button is disabled how would the `onPress` even get called?

Reply:
you're correct, I had added it as for some reason I was still able to click it while initial testing but couldn't reproduce it now so have removed it.